cmd/gossamer/main.go

```diff
@@ -174,15 +174,23 @@ func nodeInitialized(ctx *cli.Context) bool {
 	registry := path.Join(cfg.Global.DataDir, "KEYREGISTRY")
 	_, err = os.Stat(registry)
 	if os.IsNotExist(err) {
-		log.Error("[gossamer] Failed to locate KEYREGISTRY", "error", err)
+		log.Warn(
+			"[gossamer] Node has not been initialized",
+			"datadir", cfg.Global.DataDir,
+			"error", "failed to locate KEYREGISTRY file in data directory",
+		)
```

Comment:
Shouldn't we also log the value of err in the log?

Reply:
We could but the error is "stat [datadir]" and this was cleaner. I was thinking that maybe since its a warning message for the user it would be nice to clean it up.

Reply:
makes sense, I wasn't sure if the `err`  contained more context regarding the error.